backup-stores/azure/src/main/java/io/camunda/zeebe/backup/azure/FileSetManager.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.backup.azure;
+
+import com.azure.core.util.BinaryData;
+import com.azure.storage.blob.BlobContainerClient;
+import com.azure.storage.blob.specialized.BlockBlobClient;
+import io.camunda.zeebe.backup.api.BackupIdentifier;
+import io.camunda.zeebe.backup.api.NamedFileSet;
+import java.nio.file.NoSuchFileException;
+
+final class FileSetManager {
+
+  private static final String PATH_FORMAT = "contents/%s/%s/%s/%s/";
+  private final BlobContainerClient containerClient;
+
+  FileSetManager(final BlobContainerClient containerClient) {
+    this.containerClient = containerClient;
```

Comment:
:thought_balloon: I think we could even create a new container client that can only write to `contents`? Would be neat but not required.